### ISSUE TITLE
Reproducing problems

### ISSUE THREAD

**Issue by Sukeysun** [google-research/big_transfer#70]
Hi,
Thanks for your great work of these two articles!
 I am reproducing Knowledge distillation: A good teacher is patient and consistent using Flowers102 dataset. But I cannot reach the accuracy you got of Best from-scratch ResNet50(66.38%). Could you tell me the hyperparameter you used, such as the learning rate schedule, the optimizer?
Thanks.
